examples/customer-segmentation-server/src/mcp-app.ts

```diff
@@ -138,11 +143,40 @@ function buildDatasets(): Chart["data"]["datasets"] {
   });
 }
 
+// Hidden element for resolving CSS color values (reused to avoid DOM thrashing)
+let colorResolver: HTMLDivElement | null = null;
+
+// Resolve a CSS color value (handles light-dark() function)
+function resolveColor(cssValue: string, fallback: string): string {
```

Comment:
This is necessary for this particular example b/c chartJS doesn't play nicely with `light-dark` by default